src/poliastro/constants.py

```diff
@@ -31,6 +31,9 @@
 
 J2000 = time.Time('J2000', scale='tdb')
 
+G = Constant('G', 'Gravitational Constant', 6.67428e-11, 'm3 / (kg * s2)', 0.00067e-11,
```

Comment:
This is already in `astropy.constants`! http://docs.astropy.org/en/stable/constants/#module-astropy.constants